src/components/chart.vue

```diff
@@ -89,18 +92,18 @@ export default Vue.extend({
         },
         legend: {
           item: {
-            onclick: id => {
+            onclick: (id: any) => {
               if (this.focusItems.includes(id)) {
                 this.focusItems = this.focusItems.filter(v => v !== id);
               } else {
                 this.focusItems.push(id);
               }
               chart.focus(this.focusItems);
             },
-            onmouseover: id => {
+            onmouseover: (id: any) => {
```

Comment:
since this is a one-liner you don't need to have the brackets and we could just put it on one line. for `onmouseover` and `onmouseout`